src/infra/docs-lambda-index-publisher/lambda.DockerFile

```diff
@@ -31,5 +31,4 @@ RUN arch=$TARGETARCH \
     && if [ "$arch" = "amd64" ]; then arch="x64"; fi \
     && echo $TARGETOS-$arch > /tmp/rid
 
-RUN dotnet publish src/infra/docs-lambda-index-publisher -r linux-x64 -c Release
-
+RUN dotnet publish src/infra/docs-lambda-index-publisher -r $(cat /tmp/rid) -c Release
```

Comment:
Do we need to do the same here: https://github.com/elastic/docs-builder/blob/main/src/infra/docs-lambda-index-publisher/lambda.DockerFile ?

Reply:
Hehe, I unintentionally changed the wrong file..

I will keep as is, because it works.